.github/workflows/r.yml

```diff
@@ -0,0 +1,129 @@
+name: R build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: 'Spark 3.0.1 (R release)'
+            r: 'release'
+            env:
+              SPARK_VERSION: '3.0.1'
```

Comment:
Could you please  upgrade the CI to support Spark 3.1.1?
Please see our latest CI example https://github.com/apache/incubator-sedona/blob/master/.github/workflows/java.yml#L43